src/nanoemoji/nanoemoji.py

```diff
@@ -270,16 +272,30 @@ def _draw_glyph_extents(ufo: ufoLib2.Font, glyph: Glyph):
 
 
 def _glyf_ufo(ufo, color_glyphs):
+    # glyphs by reuse_key
+    glyphs = {}
+    swaps = []
     for color_glyph in color_glyphs:
-        svg_units_to_font_units = color_glyph.transform_for_font_space()
         logging.debug(
             "%s %s %s",
             ufo.info.familyName,
             color_glyph.glyph_name,
-            svg_units_to_font_units,
+            color_glyph.transform_for_font_space(),
         )
+        parent_glyph = ufo.get(color_glyph.glyph_name)
         for painted_layer in color_glyph.as_painted_layers():
-            _create_glyph(color_glyph, painted_layer)
+            # if we've seen this shape before reuse it
+            reuse_key = _inter_glyph_reuse_key(painted_layer)
```

Comment:
in theory, at least for glyf (probably not for COLR), these inter-glyph reuses need not be exactly the same `path.d` but could be normalized first in a similar way as we do for in-glyph reuses, and then referenced as components with a translation offset. I guess we could do that later.

Reply:
I think that may only make sense to add if we add transform to COLR layers. glyf can't be a color format so I don't want to optimize it all that much.